docs/release-notes/release-notes-tumbler-improve.md

```diff
@@ -0,0 +1,20 @@
+copypaste this into "release-notes" when the time comes to make a new release, then delete this file
```

Comment:
Thanks! Nice thought to reduce workload :)

Reply:
I saw that bitcoin core does this